### ISSUE TITLE
Why  conv_trans_block   stride=2, padding=1, output_padding=1

### ISSUE THREAD

**Issue by LukeLIN-web** [alertcat/unet-QAT#1]
```
def conv_trans_block(in_dim, out_dim, act_fn):
    model = nn.Sequential(
        nn.ConvTranspose2d(in_dim, out_dim, kernel_size=3, stride=2, padding=1, output_padding=1),
        nn.BatchNorm2d(out_dim),
        act_fn,
    )
    return model
```
what is the difference?

**Comment by LukeLIN-web** [comment]
We need use ` leftImg8bit `  [leftImg8bit_trainvaltest.zip (11GB)](https://www.cityscapes-dataset.com/file-handling/?packageID=3) [[md5]](https://www.cityscapes-dataset.com/md5-sum/?packageID=3)

**Comment by LukeLIN-web** [comment]
Thank you provide trained pkl pth, and then we can convert to qat
pkl saves too much things, even dataparallel object.

**Comment by LukeLIN-web** [comment]
```
32-bit Float Model:
    - Mean IoU: 0.1445
    - Per-class IoU:
        background: 0.5780
        road: 0.0000
        sky: 0.0000
        car: 0.0000
    - Ground Truth class counts:
        background: 9598961.0
        road: 5382190.0
        sky: 473073.0
        car: 929776.0
    - Prediction class counts:
        background: 16255500.0
        road: 0.0
        sky: 0.0
        car: 128500.0
    - Average Inference Time: 0.0193 seconds

Int8 Quantized Model:
    - Mean IoU: 0.1750
    - Per-class IoU:
        background: 0.4445
        road: 0.2503
        sky: 0.0000
        car: 0.0053
    - Ground Truth class counts:
        background: 9598961.0
        road: 5382190.0
        sky: 473073.0
        car: 929776.0
    - Prediction class counts:
        background: 9808398.0
        road: 6562149.0
        sky: 1762.0
        car: 11691.0
    - Average Inference Time: 0.0026 seconds
  ``` 
train 10 epoch with your qat sucess. 
I  don't know why float 32 model acc is so low.

**Comment by alertcat** [comment]
Overall Performance:
The Int8 quantized model has a higher overall mean IoU (0.1750 vs. 0.1445) because it at least predicts non-background classes (like road), which the float model completely misses.

Per-Class Trade-offs:

Background: Float model excels, but at the cost of over-predicting background pixels.
Road: The quantized model shows a clear improvement by making meaningful predictions.
Sky and Car: Both models struggle here, with IoUs close to 0.

需要增加天空和汽车类的数据。做增强学习，IOU就能提高。
